spec/factories/encounters.rb

```diff
@@ -56,7 +56,7 @@ def updated_encounters_payload(existing_encounter)
 end
 
 def associate_encounter(observable)
-  encountered_on = Encounter.generate_encountered_on(observable.recorded_at, 0)
+  encountered_on = Encounter.generate_encountered_on(observable.recorded_at, Time.find_zone(Period::REPORTING_TIME_ZONE).utc_offset)
```

Comment:
👍

Reply:
this ensures encounters are created as UTC, correct? Maybe the `generate_encountered_on` method should be responsible for that...feels like this method could just pass in the `recorded_at` ?

Reply:
The `encountered_on` date is actually converted in local (reporting) time, as understood by the rails app. This change is just doing what we do in the controller path in the test path as well.

Reply:
In the old code, the provide timestamp `observable.recorded_at` is interpreted as observed in UTC, which results in us interpreting it as the wrong day in the reporting tables.